packages/mdc-slider/README.md

```diff
@@ -100,6 +113,7 @@ CSS Class | Description
 `mdc-slider__track-fill` | Mandatory. The fill element to display where the value is.
 `mdc-slider__thumb` | Mandatory. Element containing the thumb-handle.
 `mdc-slider__thumb-handle` | Mandatory. The handle element to display where the value is.
+`mdc-slider__thumb-handle-disabled` | Mandatory for disabled state.
```

Comment:
This class should be a modifier, so `.mdc-slider__thumb-handle--disabled` _unless_ it is explicitly it's own element.

Reply:
Would be explicitly its own element, not a modifier.